docs/src/modules/java/pages/publishing-subscribing.adoc

```diff
@@ -241,7 +241,7 @@ include::example$java-spring-eventsourced-counter/src/it/java/com/example/Counte
 ----
 <1> Build a `CloudEvent` object with the 3 required attributes, respectively: `id`, `source` and `type`.
 <2> Add the subject to which the message is related, that is the `counterId`.
-<3> Set the mandatory header "Content-Type" accordingly.
+<3> Sets the mandatory header "Content-Type" accordingly.
```

Comment:
If we decide to change it, I will do this in a separate PR and change all verbs in all bullet point lists.

Reply:
Sure, I'll leave for to another PR.

Reply:
back to previous.